recipes/cpp-jwt/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
-set(CMAKE_VERBOSE_MAKEFILE TRUE)
-
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
```

Comment:
Use `conan_basic_setup(TARGETS)` for header only libraries, otherwise include dir is added so you don't realy test the target coming from module or config file.